backend/src/main/java/com/ez/ezbackend/budget/request/TransactionRequest.java

```diff
@@ -60,6 +61,9 @@ public static Transaction convertToTransaction(TransactionRequest transactionReq
   }
 
   private static void validateTransactionRequest(TransactionRequest transactionRequest) {
+    if (Strings.isNullOrEmpty(transactionRequest.getDescription())) {
```

Comment:
Hmm.. this would be more of a product question. I thought this as like a banking statement where it has description for each transaction. I will leave this for now since it is always easier to delete than to write again.